modules/vault-cluster/main.tf

```diff
@@ -227,7 +227,7 @@ resource "aws_security_group" "lc_security_group" {
     from_port       = "22"
     to_port         = "22"
     protocol        = "tcp"
-    security_groups = ["${var.ssh_security_group_id}"]
+    security_groups = ["${var.ssh_security_group_ids}"]
```

Comment:
the same as my previous comment

Reply:
Clarified [here](https://github.com/Flaconi/terraform-aws-vault/pull/11#discussion_r544991166).